# HG changeset 9668 patch
From: jbeulich@novell.com
# Node ID fc9ec6fd34002b99614d878e5a3e61259460741f
# Parent  28f9610447b277d65cb8f1b716ad55bd2aaf3e01
Subject: Add module license to xenbus_be, otherwise the module cannot be loaded.

xen-unstable changeset:   9957:85de2de6469912deb98511e6052b50d357f5682b
xen-unstable date:        Mon May  8 14:54:22 2006 +0100

References: 173143

Index: head-2006-05-19/drivers/xen/xenbus/xenbus_backend_client.c
===================================================================
--- head-2006-05-19.orig/drivers/xen/xenbus/xenbus_backend_client.c	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/drivers/xen/xenbus/xenbus_backend_client.c	2006-05-24 14:20:28.000000000 +0200
@@ -139,6 +139,7 @@ int xenbus_unmap_ring(struct xenbus_devi
 }
 EXPORT_SYMBOL_GPL(xenbus_unmap_ring);
 
+MODULE_LICENSE("Dual BSD/GPL");
 
 /*
  * Local variables:
